File: app/src/main/java/com/zf/spycamera/adapter/GalleryPagerAdapter.java
package com.zf.spycamera.adapter;

import android.content.Context;
import android.support.v4.app.Fragment;
import android.support.v4.app.FragmentManager;
import android.support.v4.app.FragmentPagerAdapter;

import java.util.List;

/**
 * Created by Allever on 18/5/16.
 */

public class GalleryPagerAdapter extends FragmentPagerAdapter{
    private Context mContext;
    private List<Fragment> mFragmentList;

    public GalleryPagerAdapter(FragmentManager fm, Context context, List<Fragment> fragmentList) {
        super(fm);
        mContext = context;
        mFragmentList = fragmentList;
    }

    @Override
    public Fragment getItem(int position) {
        return mFragmentList.get(position);
    }

    @Override
    public int getCount() {
        return mFragmentList.size();
    }

    @Override
    public float getPageWidth(int position) {
        return 0.75f;
    }
}

File: app/src/main/java/com/allever/stealthcamera/ui/adapter/GalleryPagerAdapter.kt
package com.allever.stealthcamera.ui.adapter

import android.content.Context
import androidx.fragment.app.Fragment
import androidx.fragment.app.FragmentManager

/**
 * Created by Allever on 18/5/16.
 */

class GalleryPagerAdapter(fm: FragmentManager, private val mContext: Context, private val mFragmentList: List<Fragment>) : androidx.fragment.app.FragmentPagerAdapter(fm) {

    override fun getItem(position: Int): Fragment {
        return mFragmentList[position]
    }

    override fun getCount(): Int {
        return mFragmentList.size
    }

    override fun getPageWidth(position: Int): Float {
        return 0.75f
    }
}
